src/js/components/DataTable/doc.js

```diff
@@ -159,6 +159,13 @@ export const doc = DataTable => {
       names and values which are the search text strings. This is typically
       employed so a back-end can be used to search through the data.`,
     ),
+    onSort: PropTypes.func.description(
+      `When supplied, this function will be called with an event object
+      that includes a 'sort' property containing an object with a
+      'property' property that indicates which property
+      is being sorted on and a 'direction' property that will either be
+      'asc' or 'desc'. onChange={({ sort: { property, direction }}) => {}}`,
```

Comment:
why is the example not stating `onSort` to connect the context better? I think the onChange should be replaced.

Reply:
Good catch! I've updated the documentation.